test/commonjs/basic.js

```diff
@@ -69,7 +69,7 @@ app.ready(function (err) {
   })
 
   app.inject({
-    url: '/semiautomatic/items'
+    url: '/manualprefix/semiautomatic/items'
```

Comment:
Is this change necessary due to the change in the fixture, or is this a breaking change?

Can you please not alter existing tests?

Reply:
Let's avoid a breaking change, thanks.